bank-sdk/example-app/src/androidTest/java/net/gini/android/bank/sdk/exampleapp/ui/screens/MainScreen.kt

```diff
@@ -0,0 +1,46 @@
+package net.gini.android.bank.sdk.exampleapp.ui.screens
+
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.isDisplayed
+import net.gini.android.bank.sdk.exampleapp.R
+
+class MainScreen {
+
+    fun assertWelcomeTitle(): MainScreen {
+        onView(withId(R.id.tv_welcomeToGini)).check(matches(isDisplayed()))
+        return this
+    }
+
+    fun assertDescriptionTitle(): MainScreen {
+        onView(withId(R.id.tv_exampleOfPhotoPayment)).check(matches(isDisplayed()))
```

Comment:
[Suggestion] As well as the test above, this looks like it checks for a view with id `R.id.exampleOfPhotoPayment` is displayed. Could we change the title (or what the test does) to reflect what is being checked?

Thank you!

Reply:
Sure, I will change the title of test for this one as well..